src/java/org/apache/cassandra/config/GuardrailsOptions.java

```diff
@@ -345,6 +345,20 @@ public void setReadBeforeWriteListOperationsEnabled(boolean enabled)
                                   x -> config.read_before_write_list_operations_enabled = x);
     }
 
+    @Override
+    public boolean getAllowFilteringEnabled()
+    {
+        return config.allow_filtering_enabled;
+    }
+
+    public void setAllowFilteringEnabled(boolean enabled)
```

Comment:
`@Override`?

Reply:
Only the `get` methods in Guardrails are overridden from the `GuardrailsConfig` class .